src/main/java/org/opentripplanner/netex/mapping/FlexStopsMapper.java

```diff
@@ -117,25 +131,22 @@ AreaStop mapFlexArea(FlexibleStopPlace flexibleStopPlace, FlexibleArea area) {
   /**
    * Allows pickup / drop off at any regular Stop inside the area
    */
-  GroupStop mapStopsInFlexArea(FlexibleStopPlace flexibleStopPlace, FlexibleArea area) {
-    GroupStopBuilder result = GroupStop
-      .of(idFactory.createId(flexibleStopPlace.getId()))
-      .withName(new NonLocalizedString(flexibleStopPlace.getName().getValue()));
-
+  List<RegularStop> findStopsInFlexArea(FlexibleArea area) {
+    List<RegularStop> stops = new ArrayList<>();
     Geometry geometry = mapGeometry(area);
     if (geometry == null) {
-      return null;
+      return List.of();
```

Comment:
I can't tell what the significance of this change is. @hannesj would you be able to shine some light?

Reply:
This makes the build process not crash, if there is an stop area with an invalid geometry. Otherwise iteration of null would cause an NPE.

Reply:
Yeah, so trying to find the stops within an area with invalid geometry will result in an empty list of stops, whereas before it would cause an NPE and stop the whole build? Do we even allow FlexibleArea's with invalid geometry to get this far or should it be stopped earlier in the process?

Reply:
This is the place where the `OpenGisMapper` is called, and the first place where geometry issues, as long as the geometry can be deserialized, are uncovered.